lib/versions-complete.xml

```diff
@@ -2584,6 +2584,18 @@
 					</license>
 				</licenses>
 			</library>
+			<library>
+				<file-name>com.liferay.object.service.jar!groovy.jar</file-name>
```

Comment:
@jonathanmccann People may often forget to build-lib-versions when changing third-party lib versions, and then these will sneak into the build-lib-versions of other lib version changes. Could you only commit the changes related to the Spring upgrade?